ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2656,6 +2655,12 @@ def get_pki_version(host):
         raise ValueError("get_pki_version: pki is not installed")
 
 
+def get_openldap_client_version(host):
+    """Get openldap-clients version on remote host"""
+    version = host.run_command('rpm -q openldap-clients')
+    return parse_version(version)
+
```

Comment:
Please see the code from get_healthcheck_version for an example of how to extract the version.